index.html

```diff
@@ -0,0 +1,106 @@
+                    <li>
+                        <strong>1914</strong>
+                         - Born in Cresco, lowa
+                    </li>
+                    <li>
+                        <strong>1933</strong>
+                        - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"
+                    </li>
+                    <li> 
+                        <strong>1935</strong>
+                        - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+                    </li>
+                    <li>
+                        <strong>1937</strong>
+                        - Finishes university and takes a job in the US Forestry Service
+                    </li>
+                    <li>
+                        <strong>1938</strong>
+                        - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.
+                    </li>
+                    <li>
+                        <strong>1941</strong>
+                        - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.
+                    </li>
+                    <li>
+                        <strong>1942</strong>
+                        - Receives a Ph.D. in Genetics and Plant Pathology
+                    </li>
+                    <li>
+                        <strong>1944</strong>
+                        - Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000 different strains of disease resistent wheat - including different varieties for each major climate on Earth.
+                    </li>
+                    <li>
+                        <strong>1945</strong>
+                        - Discovers a way to grown wheat twice each season, doubling wheat yields
+                    </li>
+                    <li>
+                        <strong>1953</strong>
+                        - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed, creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+                    </li>
+                    <li>
+                        <strong>1962</strong>
+                        - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help mitigate mass starvation due to a rapidly expanding population
+                    </li>
```

Comment:
It is very clear and good coding standard, if you manage your codes in a same line with indentation.

Reply:
- [ ] Hi @abdullah97-pro 👋, Well done on using inline comments and giving a reason. 👍
- Kindly use some kind words such as so that it doesn't seem like an order but rather seems like a request. As example: `It would be better to manage your codes in the same line with indentation, as it is a very clear and good coding standard.`